docs/content/modeling/getting-started.mdx

```diff
@@ -354,28 +336,24 @@ Now that we have a list of object types we can start defining them using the <Up
   }}
 />
 
-:::info Caution
-You're now in the process of building a version you can use. The model above is not yet a valid authorization model accepted by <ProductName format={ProductNameFormat.ShortForm}/>.
-:::
-
 :::info Important
-In a few cases other users can be part of determining whether an action can be performed on an action or not. Social media is an example of this "a user can comment on a picture if they are a friend of the user that published it".
+Though you're in the process of building a version you can use, the model above is not yet a valid authorization model accepted by <ProductName format={ProductNameFormat.ShortForm}/>.
+
+In a few cases, other users can determine whether one action can be performed on another action. For example, on social media, sometimes a user can comment on a picture if they are a friend of the user that published it.
```

Comment:
I don't think this whole note is helpful here, let's remove it. It used to be back in back in the day where we didn't require a type for an object

Reply:
Fair enough!